pkgs/misc/drivers/xow/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub, libusb, gitMinimal }:
+
+stdenv.mkDerivation rec {
+  pname = "xow";
+  version = "0.2";
+
+  src = fetchFromGitHub {
+    owner = "medusalix";
+    repo = "xow";
+    rev = "v${version}";
+    sha256 = "03ajal91xi52svzy621aa4jcdf0vj4pqd52kljam0wryrlmcpbr3";
+  };
+
```

Comment:
In `postPatch` we should fix the path in the service file:
https://github.com/medusalix/xow/blob/897eee07351f88a04eb16a02b8bca0b91f91cad4/xow.service#L5

Also, an issue should be opened upstream.

Reply:
What's a better path for the executable in the service file in this case?

Reply:
The path should be based on the `INSTALL_PATH` passed to `make`, instead of the hardcoded `/usr/local/bin`.

Reply:
You mean `make` should change the service file to adapt the path and install that to the proper location?

Reply:
Yes. This is usually done by having `foo.service: foo.service.in` target and creating `foo.service` from `foo.service.in` by replacing the template values.

Reply:
Okay. Instead of patching this systemd and Makefile file in lots of places, I'd like to open an issue upstream and see if we can get the whole file + installation process fixed for the next xow release.

Reply:
Any improvements to the build process are welcome, though I'd like to keep the Makefile as simple as possible (part of the reason I didn't consider Autotools).